ci-conda.Dockerfile

```diff
@@ -233,8 +233,17 @@ wget https://github.com/cli/cli/releases/download/v${GH_CLI_VER}/gh_${GH_CLI_VER
 tar -xf gh_*.tar.gz
 mv gh_*/bin/gh /usr/local/bin
 rm -rf gh_*
+
+# Install OpenTelemetry instrumentation
+pip install opentelemetry-distro[otlp] opentelemetry-exporter-prometheus
+opentelemetry-bootstrap -a install
+curl -L -o "otel-cli-${CPU_ARCH}.tar.gz" https://github.com/equinix-labs/otel-cli/releases/download/v0.4.5/otel-cli_0.4.5_linux_${CPU_ARCH}.tar.gz
+tar -zxf  "otel-cli-${CPU_ARCH}.tar.gz"
```

Comment:
I see a few more files inside this tarball that may need to be cleaned up after extraction (see screenshot below)

![image](https://github.com/user-attachments/assets/25b3e6ac-4fad-43d3-93cb-fcea639ba697)

to avoid having to clean these up, you can specify which files you want to extract like this:

```sh
# specify the path of the files you wish to extract from the tarball after the tarball filename
tar -zxf "otel-cli-${CPU_ARCH}.tar.gz" otel-cli
```

this will ensure that the `LICENSE`, `README.md`, and `CHANGELOG.md` files don't get extracted and clutter up the filesystem.

if you want to get real fancy, you can use a pipe to avoid saving the tarball to the filesystem entirely:

```sh
wget -qO- https://github.com/equinix-labs/otel-cli/releases/download/v0.4.5/otel-cli_0.4.5_linux_${CPU_ARCH}.tar.gz | tar xzf - -C /usr/local/bin otel-cli
```

then you can skip the subsequent `rm -rf` command altogether.

source: https://unix.stackexchange.com/q/85194